build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,30 @@
+FROM registry.access.redhat.com/ubi8/openjdk-11
```

Comment:
We should directly base it on the new java LTS version (https://openjdk.java.net/projects/jdk/17/) published 14.08.2021.

Reply:
Might take a while until RedHat adopts that, but would be nice to have whenever they are ready :)